### ISSUE TITLE
Custom cache folder question: Where is the cache supposed to be?

### ISSUE THREAD

**Issue by Larx** [MediaBrowser/Emby#1152]
I try to keep my cache in a custom folder, let's call it
/raid/emby/cache. 

I therefore entered 
/raid/emby/
as custom cache path. (This is because from a former try I gathered that emby creates the subfolder "cache" by itself in the path given here, which seems to be right for the first start of emby with the custom cache folder.)

This works for a time, but after a restart emby starts to fill a new subfolder
/raid/emby/cache/cache.

The entry in system.xml is still
<CachePath>/raid/emby/</CachePath>
and also on the web interface, server settings, this path is displayed.

However, using the web interface and looking into "server overview, paths", I can read:
Cache: /raid/emby/cache.

What am I doing wrong here, is this expected behaviour, or does emby always create a new "cache" subfolder in the given cache folder?

**Comment by Larx** [comment]
I tried it out:
Defining a custom folder /a/:
Cache is in /a/cache/ after the first restart.
Cache starts to fill up in /a/cache/cache after the next restart

Seems like a bug to me...

**Comment by LukePulverenti** [comment]
Where are you saving this setting? I have not been able to reproduce that.

**Comment by Larx** [comment]
1. I set the custom cache path in the web interface (German language, so I try to guess the english words): Server -> settings -> advanced -> cache path.
I set it to
/raid/emby/
(which is still the path is also contained in the config file "system.xml")

2. In the beginning emby started to fill the folder
/raid/emby/cache/
with its cache data. The trailing "/cache/" was added automatically, it seems.

3. But currently, I guess after the next server restart, the text field I filled in step 1 reads
/raid/emby/cache/
despite the fact that I entered /raid/emby/ there (as I said, system.xml still contains this original value I entered)

4. If I use the web interface again and go to Server -> overview -> paths -> cache, I see
Cache: /raid/emby/cache/cache/

In fact, this is the drawer emby seems to fill currently with its cache data. 

It seems that the multipling of "/cache/" subfolders stops here, at least currently no more new cache-folders are created.
